lib/elixir/pages/anti-patterns/code-anti-patterns.md

```diff
@@ -371,6 +371,135 @@ There are few known exceptions to this anti-pattern:
 
   * If you are the maintainer for both `plug` and `plug_auth`, then you may allow `plug_auth` to define modules with the `Plug` namespace, such as `Plug.Auth`. However, you are responsible for avoiding or managing any conflicts that may arise in the future
 
+## Non-assertive patterns
```

Comment:
The word "pattern" might be slightly confusing due to "pattern-matching" (which in this case it's precisely not).
How about `Non-assertive code` or `Non-assertive style` (both inspired from this [article](https://dashbit.co/blog/writing-assertive-code-with-elixir))?

Reply:
I was playing intentionally on the ambiguity. Although I would say, even in the poor case, it is pattern matching, because `foo = ...` is a pattern matching (but one with no assertion).

Reply:
I would be happy to rename this to `Non-assertive pattern matching` though.

Reply:
Makes sense, I agree that `Non-assertive pattern matching` would clear up that ambiguity if this is the intention.

Reply:
Agreed, *Non-assertive pattern matching* sounds great.

Reply:
```suggestion
## Non-assertive pattern matching
```